src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjectTests.cs

```diff
@@ -2568,12 +2568,14 @@ public unsafe void TestAccessibleObjectFromPoint_ComboBox()
             Assert.Equal(7, parent.accChildCount);
         }
 
-        [WinFormsFact]
-        public void AccessibleObject_GetPropertyValue_ControlTypeProperty_ReturnsNull()
+        [WinFormsTheory]
+        [InlineData((int)UiaCore.UIA.ControlTypePropertyId)]
+        [InlineData((int)UiaCore.UIA.AutomationIdPropertyId)]
+        public void AccessibleObject_GetPropertyValue_ReturnsNull(int propertyId)
```

Comment:
I don't like this name, because `GetPropertyValue` returns `null` not for all cases. You can rename like that, for example.
```suggestion
        public void AccessibleObject_GetPropertyValue_ReturnsNull_ForSeveralProperties(int propertyId)
```

Reply:
I believe that AccessibleObject_GetPropertyValue_ReturnsNull_IfExpected should be good.